### PR TITLE
Re-establish support for alpha on markers (InspectDR).

### PR DESCRIPTION
**Pull request by ma-laforge**

### CODE REVIEW

File: src/backends/inspectdr.jl [hunk 1]
```diff
@@ -22,6 +22,7 @@ const _inspectdr_attr = merge_with_base_supported([
     :foreground_color_legend, :foreground_color_title,
     :foreground_color_axis, :foreground_color_border, :foreground_color_guide, :foreground_color_text,
     :label,
+#    :seriescolor, :seriesalpha,
```
Comment (daschw) [inline src/backends/inspectdr.jl]: `linecolor`, `fillcolor`, etc. fall back to `seriescolor`. The same is true for `...alpha`. So `seriescolor` and `seriesalpha` are supported.

**Comment by ma-laforge** [comment]
Appveyor build failiures do not appear to be caused by this PR.  It appears to be an issue with package versions.

Do I need to do anything?

**Comment by daschw** [comment]
No, I'd uncomment the line with seriescolor and seriesalpha to avoid possible warnings (not sure, did not test), but else it looks good to merge.

**Comment by ma-laforge** [comment]
Ok.  I thought you were actually making the change yourself.  I did not notice this was merely a comment.

I will update the PR soon.

**Comment by daschw** [comment]
Thanks a lot, and sorry, I could have been clearer.

**Pull request merged**